packages/create-sitecore-jss/src/templates/angular/src/app/lib/config.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Represents the type of config object available within the generated /environments/environment.js
+ */
+export interface JssConfig extends Record<string, string | boolean | undefined> {
+  production?: false;
+  sitecoreApiKey?: string;
+  sitecoreApiHost?: string;
+  sitecoreSiteName?: string;
+  defaultLanguage?: string;
+  graphQLEndpoint?: string;
+  graphQLEndpointPath?: string;
+  <% if (!locals.xmcloud) { -%>
+  layoutServiceConfigurationName?: string;
```

Comment:
Same here, add `sitecoreLayoutServiceConfig`?
```suggestion
  sitecoreLayoutServiceConfig?: string;
  layoutServiceConfigurationName?: string;
```